Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'surveyor', :git => 'https://github.com/caboteria/surveyor.git', branch: 'ra
 gem 'therubyracer', '0.10.2', :platforms => :ruby
 gem 'uglifier', '>= 1.0.3'
 gem 'will_paginate'
+gem 'omniauth-cas'
```

Comment:
As far as I can tell omni-cas is the only existing solution at this point. Many of the libraries that are used in SPARC are many years old. If this is the metric that code can be contributed we will be stuck in adding new solutions that use existing code that support the libraries that are used, such as omni-auth. Are you suggesting that we must write a new omni-auth plugin of our own and then get then contribute that?